### ISSUE TITLE
Why does some of the feature activation plotting leave out tokens? Investigate this.

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#87]
I should have grey highlighting for 0.0 affected tokens, as I do on the logits side of things.

**Comment by DavidUdell** [comment]
Fixed. Was missing a range of possible activations in the plot, oops.
